Source/WebCore/accessibility/AccessibilityList.cpp

```diff
@@ -178,7 +176,7 @@ AccessibilityRole AccessibilityList::determineAccessibilityRoleWithCleanChildren
                 if (!hasVisibleMarkers && (renderListItem->style().listStyleType().type != ListStyleType::Type::None || renderListItem->style().listStyleImage() || childHasPseudoVisibleListItemMarkers(renderListItem->element())))
                     hasVisibleMarkers = true;
                 listItemCount++;
-            } else if (node && node->hasTagName(liTag)) {
+            } else if (node && WebCore::elementName(node.get()) == ElementName::HTML_li) {
```

Comment:
Could use the Node& version of elementName here to avoid duplicating the null-check